src/basic/conf-files.c

```diff
@@ -469,3 +469,30 @@ int conf_file_read(
 
         return r;
 }
+
+int find_portable_profile(const char *name, const char *unit, char **ret_path) {
```

Comment:
is this really the right place for this?

i'd probably create portable-util.[ch] for this. sounds specific enough to deserve this